### PR TITLE
Request creation

### PR DESCRIPTION
**Pull request by nilscox**

### CODE REVIEW

File: apps/web/src/app.tsx [hunk 1]
```diff
@@ -83,7 +87,15 @@ const Routing: Component = () => {
       <Route path="/onboarding" component={OnboardingPage} />
       <Route path="/members" component={MembersPage} />
       <Route path="/members/:memberId" component={MemberPage} />
-      <Route path="/requests" component={RequestsPage} />
+
+      <FeatureFlag
```
Comment (Bopzor) [inline apps/web/src/app.tsx:91]: **praised**

This is awesome. So simple yet so cool!
I add to trick by commenting and importing correct `ResquestPage` in the other branch.
Much cleaner :)

File: apps/web/src/components/feature-flag.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+type FeatureFlagProps = {
+  feature: Feature;
+  fallback?: JSX.Element;
+  children?: JSX.Element;
+};
+
+export function FeatureFlag(props: FeatureFlagProps) {
+  const features = JSON.parse(localStorage.getItem('features') ?? '{}');
+  console.log(features[props.feature]);
```
Comment (Bopzor) [inline apps/web/src/components/feature-flag.tsx:16]: **question/nitpick**

Is this still needed?

**Comment by Bopzor** [comment]
We should improve the caching, because when creating a request and going back to the requests list, the list is not up-to-date.

File: apps/web/src/components/feature-flag.tsx (hunk 2 above)
Comment (nilscox) [reply apps/web/src/components/feature-flag.tsx:16]: I don't think it is, I rushed this part because my train was arriving haha

Comment (Bopzor) [reply apps/web/src/components/feature-flag.tsx:16]: We should add an eslint rule about this one :nerd_face:

**Review by Bopzor — APPROVED** [review]

File: apps/web/src/components/feature-flag.tsx (hunk 2 above)
Comment (nilscox) [reply apps/web/src/components/feature-flag.tsx]: https://github.com/Bopzor/sel/pull/31/commits/a352e889cd6d36ddd0b05b7cbaa8269c26a19108#diff-0b67f0bda338ae0b05d8b4024b2e5915d659d1d191900217012909f8633a7ebeR40

**Comment by nilscox** [comment]
Rich text placeholder color in dark mode

**Pull request merged**